src/InputCarsException.js

```diff
@@ -0,0 +1,44 @@
+import { errorCarMessage } from "./message.js";
+
+class InputCarsException {
+  constructor(cars) {
+    this.cars = cars;
+  }
+
+  checkIsNameEmpty(cars) {
+    if (cars === "") throw new Error(errorCarMessage.INVALID_CAR_NAME_EMPTY);
+  }
+
+  checkNameLength(cars) {
+    for (let i = 0; i < cars.length; i++) {
+      if (cars[i].length > 5)
+        throw new Error(errorCarMessage.INVALID_CAR_NAME_LENGTH);
+    }
+  }
+
+  checkDuplicate(cars) {
+    let checkDuplicateSet = new Set();
+    for (const car of cars) {
+      if (checkDuplicateSet.has(car)) {
+        throw new Error(errorCarMessage.INVALID_CAR_NAME_DUPLICATE);
+      }
+      checkDuplicateSet.add(car);
+    }
+  }
+
+  checkIsNameSpacedBetween(cars) {
+    for (let i = 0; i < cars.length; i++) {
```

Comment:
에어비앤비 스타일 가이드에서는 다음과 같은 항목이 있는데요,
[13.6](https://github.com/airbnb/javascript#variables--unary-increment-decrement) Avoid using unary increments and decrements (++, --). eslint [no-plusplus](https://eslint.org/docs/rules/no-plusplus)

> Why? Per the eslint documentation, unary increment and decrement statements are subject to automatic semicolon insertion and can cause silent errors with incrementing or decrementing values within an application. It is also more expressive to mutate your values with statements like num += 1 instead of num++ or num ++. Disallowing unary increment and decrement statements also prevents you from pre-incrementing/pre-decrementing values unintentionally which can also cause unexpected behavior in your programs.

요약하자면 ++와 같은 단항 증가 연산자를 사용할 경우 예기치 못한 오류가 발생할 수 있다는 점입니다.
따라서 `for (let i = 0; i < cars.length; i += 1)` 과 같은 사용을 권고하고 있어요!

Reply:
오 이것도 처음 안 사실이네요! ++나 --를 쓰기 좋아했는데.. 이제부턴 주의해야겠어요!